### ISSUE TITLE
Error in 3.1.16

### ISSUE THREAD

**Issue by flovilmart** [mongodb-js/runner#58]
hey guys, seems that the 3.1.16 release has a slightly embarrassing problem that was not here in 3.1.15

``` 
/home/travis/build/ParsePlatform/parse-server/node_modules/mongodb-runner/node_modules/mongodb-tools/node_modules/mongodb-core/lib/topologies/server.js:781
      catch(err) { process.nextTick(function() { throw err}); }
                                                 ^
Error: channel closed
    at process.target.send (internal/child_process.js:510:16)
    at /home/travis/build/ParsePlatform/parse-server/node_modules/mongodb-runner/lib/worker.js:250:15
    at null.<anonymous> (/home/travis/build/ParsePlatform/parse-server/node_modules/mongodb-runner/node_modules/mongodb-tools/lib/tools/server_manager.js:230:11)
    at emitOne (events.js:77:13)
    at emit (events.js:169:7)
    at /home/travis/build/ParsePlatform/parse-server/node_modules/mongodb-runner/node_modules/mongodb-tools/node_modules/mongodb-core/lib/topologies/server.js:408:23
    at /home/travis/build/ParsePlatform/parse-server/node_modules/mongodb-runner/node_modules/mongodb-tools/node_modules/mongodb-core/lib/topologies/server.js:780:13
    at Callbacks.emit (/home/travis/build/ParsePlatform/parse-server/node_modules/mongodb-runner/node_modules/mongodb-tools/node_modules/mongodb-core/lib/topologies/server.js:95:3)
    at null.messageHandler (/home/travis/build/ParsePlatform/parse-server/node_modules/mongodb-runner/node_modules/mongodb-tools/node_modules/mongodb-core/lib/topologies/server.js:249:23)
    at Socket.<anonymous> (/home/travis/build/ParsePlatform/parse-server/node_modules/mongodb-runner/node_modules/mongodb-tools/node_modules/mongodb-core/lib/connection/connection.js:265:22)
```

**Comment by imlucas** [comment]
Sorry about that @flovilmart!  not sure how that one slipped through our tests. published fix in mongodb-runner@3.1.17

**Comment by flovilmart** [comment]
Excellent!
I'll update parse server
